README.md

```diff
@@ -13,50 +13,120 @@
+
+### Setup -- Setting up Sandbox and Deploying NFT Contract
+
+This includes launching our sandbox, loading our wasm file and deploying that wasm file to the sandbox environment.
+
+```rust
 
 #[tokio::test]
-async fn test_deploy_and_view() -> anyhow::Result<()> {
+async fn test_nft_contract() -> anyhow::Result<()> {
     let worker = workspaces::sandbox().await?;
+    let wasm = std::fs::read(NFT_WASM_FILEPATH)?;
+    let contract = worker.dev_deploy(&wasm).await?;
+```
+where
+* `anyhow` - A crate that deals with error handling, making it more robust for developers.
+* `worker` - Our gateway towards interacting with our sandbox environment.
+* `contract`- The deployed contract on sandbox the developer interacts with.
 
-    let contract = worker.dev_deploy(include_bytes!("path/to/file.wasm"))
-        .await
-        .expect("could not dev-deploy contract");
+Then we'll go directly into making a call into the contract, and initialize the NFT contract's metadata:
+```rust
+    let outcome = contract
+        .call(&worker, "new_default_meta")
+        .args_json(json!({
+            "owner_id": contract.id(),
+        }))?
+        .transact()
+        .await?;
 
-    let result: String = contract.call(&worker, "function_name")
-        .args_json(serde_json::json!({
-            "some_arg": "some_value",
+    // outcome contains data like logs, receipts and transaction outcomes.
+    println!("new_default_meta outcome: {:#?}", outcome);
+```
+
+Afterwards, let's mint an NFT via `nft_mint`. This showcases some extra arguments we can supply, such as deposit and gas:
+
+```rust
+    let deposit = 10000000000000000000000;
+    let outcome = contract
+        .call(&worker, "nft_mint")
+        .args_json(json!({
+            "token_id": "0",
+            "token_owner_id": contract.id(),
+            "token_metadata": {
+                "title": "Olympus Mons",
+                "dscription": "Tallest mountain in charted solar system",
+                "copies": 1,
+            },
         }))?
+        .deposit(deposit)
+        // nft_mint might consume more than default gas, so supply our own gas value:
+        .gas(near_units::parse_gas("300 T"))
+        .transact()
+        .await?;
+
+    println!("nft_mint outcome: {:#?}", outcome);
+```
+Then later on, we can view our minted NFT's metadata via our `view` call into `nft_metadata`:
+```rust
+    let result: serde_json::Value = contract
+        .call(&worker, "nft_metadata")
         .view()
         .await?
         .json()?;
 
-    assert_eq!(result, "OUR_EXPECTED_RESULT");
+    println!("--------------\n{}", result);
+    println!("Dev Account ID: {}", contract.id());
     Ok(())
 }
 ```
 
 ## Examples
-Some examples can be found in `examples/src/*.rs` to run it standalone.
+Some further examples can be found in `examples/src/*.rs` to be ran standalone.
```

Comment:
More standalone examples can be found in `examples/src/*.rs`

Reply:
as you can see, I'm no English major haha